deploy-board/deploy_board/webapp/cluster_view.py

```diff
@@ -332,6 +334,8 @@ def promote_image(request, image_id):
 def demote_image(request, image_id):
     try:
         baseimages_helper.demote_image(request, image_id)
+    except IllegalArgumentException as e:
```

Comment:
how do we raise this exception IllegalArgumentException ?

Reply:
Yeah we should raise in deploy-board/deploy_board/webapp/helpers/base_client.py